storage/inchi/src/main/java/org/openscience/cdk/inchi/InChIGenerator.java

```diff
@@ -350,7 +352,12 @@ private void generateInchiFromCDKAtomContainer(IAtomContainer atomContainer, boo
                 InchiAtom at2 = atomMap.get(surroundingAtoms[2]);
                 InchiAtom at3 = atomMap.get(surroundingAtoms[3]);
                 InchiStereoParity p;
-                if (stereoType == Stereo.ANTI_CLOCKWISE) {
+
+                //If all Tetrahedral stereo are marked as AND1 (&1), which means
+                //this molecule is this stereoisomer "and" the enantiomer, set parity to undefined.
+                if(stereoElem.getGroupInfo() == IStereoElement.GRP_RAC1){
+                    p = InchiStereoParity.UNDEFINED;
```

Comment:
Is that correct logic? Note you are only checking a single stereocenter here and not all of them.

Reply:
@fbaensch-beilstein I'll happily hand this one over to you :)

Reply:
I'm sorry about that. The logic is correct, it identifies the ‘and’ stereo centres. The comment is misleading (copy&paste error). 
The reason for the logic is for example the structure that is tested in the new `andEnantiomer_test` in `InChIGeneratorTest`.

Reply:
OK but the logic is still wrong... this is only testing if the group is &1.. but it could be &2, &3 etc. It might be reasonable to throw an error since I think InChI just can't handle it.

Using CXSMILES (MOLfile also possible) these are all the same:

```
CCC[C@H](O)CC[C@H](N)CCCO |r|
CCC[C@H](O)CC[C@H](N)CCCO |&1:3,7|
CCC[C@H](O)CC[C@H](N)CCCO |&2:3,7|
```

It may also be the case that only some atoms are in the group

```
CCC[C@H](O)CC[C@H](N)CCCO |&1:3|
CCC[C@H](O)CC[C@H](N)CCCO |&3:3| 
CCC[C@H](O)CC[C@H](N)CCCO |&1:7|
CCC[C@H](O)CC[C@H](N)CCCO |&3:7|
```

they may also be in the relative group

```
CCC[C@H](O)CC[C@H](N)CCCO |o1:3|
CCC[C@H](O)CC[C@H](N)CCCO |o1:7|
```

Reply:
Sorry posted to soon, but basically I think the only one InChI somewhat handles (I think it makes it a NonStd InChI) is global racemic (MDL Chiral Flag = 0)

```
CCC[C@H](O)CC[C@H](N)CCCO |r|
```

Reply:
I need to spend more time looking at this PR but in the ANDenantiomer.mol the hashed wedge is the wrong way to what we (and InChI expect). We follow IUPAC recommendations: https://publications.iupac.org/pac/pdf/2008/pdf/8002x0277.pdf
https://publications.iupac.org/pac/2006/pdf/7810x1897.pdf

There is a whole section on it but the key bit...

> The least disruptive recommendation is to reject a perspective-based approach, and rather to interpret the bonds as more-abstract graphical objects. ... , and a hashed wedged bond should be interpreted as projecting **below** the plane of the paper, also from the **narrow end to the wide end** (with the wide end being further away from the viewer). To summarize, the narrow end of solid wedged bond or a hashed wedged bond always should point towards the stereogenic center to which it applies.

Reply:
Thanks for the literature tips!

I agree with you that this is the wrong way to present this bond. 
The example comes from the RInChI repo and was isolated by us.

![image](https://github.com/user-attachments/assets/50f47ff8-243f-49cf-a5db-ba6d7076275f)

InChI handles this stereo center as undefined. And the if-statement was inserted in order to achieve a congruent result with InChI/RInChI and don't set this stereo element to odd or even. But you are right, the stereo element could also be in group 2 or 3 or so on. What would you expect here? Extend the if-statement or deviate from the native implementation or something completely different? I'd really like to hear your thoughts on this.